vue3-demo/home/webpack.config.js

```diff
@@ -0,0 +1,88 @@
+const path = require("path");
+const { VueLoaderPlugin } = require("vue-loader");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const { ModuleFederationPlugin } = require("webpack").container;
+module.exports = (env = {}) => ({
+  mode: "development",
+  cache: false,
+  devtool: "source-map",
+  optimization: {
+    minimize: false,
+  },
+  target: "web",
+  entry: path.resolve(__dirname, "./src/main.js"),
+  // output: {
```

Comment:
lets delete any commented out code and clean this up a little now that its final